controller/space_areas_test.go

```diff
@@ -154,14 +74,47 @@ func searchInAreaSlice(searchKey uuid.UUID, areaList *app.AreaList) *app.Area {
 	return nil
 }
 
-func createSpaceArea(name string, desc *string) *app.CreateSpaceAreasPayload {
+func (rest *TestSpaceAreaREST) TestListAreas() {
+	t := rest.T()
+	resource.Require(t, resource.Database)
 
-	return &app.CreateSpaceAreasPayload{
-		Data: &app.Area{
-			Type: area.APIStringTypeAreas,
-			Attributes: &app.AreaAttributes{
-				Name: &name,
-			},
-		},
+	/*
+		Space X --> Area 2 ---> Area 21-0
+	*/
+	var createdAreaUuids []uuid.UUID
+
+	parentArea := createSpaceAndArea(t, rest.db)
+	createdAreaUuids = append(createdAreaUuids, parentArea.ID)
+
+	parentID := parentArea.ID
+	name := "Area 2000A"
```

Comment:
@sbose78 if there are not restrictions on the naming of the Area, we could reuse the test name  `Area rest.T().Name()` instead of adding an arbitrary value. That could also help us when checking the data in the db.